### ISSUE TITLE
Add support for custom binary operators

### ISSUE THREAD

**Issue by erboose** [End-to-end-provenance/RDataTracker#240]
R allows users to define custom binary operators but such operators are not currently supported by RDataTracker.  For example:

# R script
`%test%` = function (a, b) a * b
2 %test% 4

When this script is executed with ddg.run, the following error is returned:

"Error in parse(text = var): <text>:1:1: unexpected SPECIAL\n1: %test%\n    ^\n"

**Comment by blernermhc** [comment]
When I run this without RDataTracker, I get this message:

```
> source('~/tmp/ddg_bugs/CustomOperator/CustomOperator.R')
Error in source("~/tmp/ddg_bugs/CustomOperator/CustomOperator.R") : 
  ~/tmp/ddg_bugs/CustomOperator/CustomOperator.R:1:1: unexpected SPECIAL
1: %test%
    ^
```
We should double-check the syntax on custom operators.

**Comment by blernermhc** [comment]
Oh, never mind.  There are supposed to be \` around %test% when it is defined but Markdown removed them.  It should be:
```
`%test%` = function (a, b) a * b
2 %test% 4
```
This does work in R but not with RDataTracker.

**Comment by blernermhc** [comment]
Fixed in 4299d53
